server/src/main/java/com/objectcomputing/checkins/services/memberprofile/currentuser/CurrentUserController.java

```diff
@@ -0,0 +1,69 @@
+package com.objectcomputing.checkins.services.memberprofile.currentuser;
+
+import com.objectcomputing.checkins.services.memberprofile.MemberProfile;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.annotation.Controller;
+import io.micronaut.http.annotation.Get;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.authentication.Authentication;
+import io.micronaut.security.rules.SecurityRule;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import java.net.URI;
+import java.util.*;
+
+@Controller("/services/member-profile/current")
+@Secured(SecurityRule.IS_AUTHENTICATED)
+@Tag(name = "current user")
+public class CurrentUserController {
+
+    @Inject
+    CurrentUserServices currentUserServices;
+
+    /**
+     * Get user details from Google authentication
+     *
+     * @param authentication {@link Authentication} or null
+     * @return {@link HttpResponse<CurrentUserDTO>}
+     */
+    @Get
+    public HttpResponse<CurrentUserDTO> CurrentUser(@Nullable Authentication authentication) {
```

Comment:
method names shouldn't start with a capital letter

Reply:
+1

Reply:
gaaah.. idk how I missed this. I've updated it.